pnc-core/src/test/java/org/jboss/pnc/core/test/buildCoordinator/ProjectBuilder.java

```diff
@@ -49,6 +50,7 @@ public static JavaArchive createDeployment() {
         JavaArchive jar = ShrinkWrap.create(JavaArchive.class)
                 .addClass(Configuration.class)
                 .addClass(Environment.Builder.class)
+                .addClass(AuthenticationProvider.class)
```

Comment:
I would like to have a separate integration test for authentication/authorization.